xonotic.patch

```diff
@@ -81,6 +81,7 @@ index 50d829e..85fbe91 100644
 +SUFFIX ?= $(shell if [ -d .git ]; then echo git; elif [ x"$(BINARY)" = x"yes" ]; then echo zip-binary; else echo zip-source; fi)
```

Comment:
What about [this patch](https://github.com/jjardon/org.xonotic.Xonotic/commit/86f53edebff805a26266cb4c24e3a691bea678b6)? Why not using that approach instead of your patch?